src/test/scala/uk/gov/nationalarchives/tdr/validation/MetadataValidationSpec.scala

```diff
@@ -0,0 +1,231 @@
+    val error = metadataValidation.validateClosureMetadata(input)
+    error should be(List(Error(closureType, CLOSURE_METADATA_EXISTS_WHEN_FILE_IS_OPEN)))
+  }
+
+  "validateClosureMetadata" should "return an error if closure metadata has invalid values" in {
+
+    val dependentMetadataCriteria = List(
+      MetadataCriteria("Property1", Boolean, true, false, false, List("yes", "no"), None, None),
+      MetadataCriteria("Property2", Text, true, false, false, Nil, None, None),
+      MetadataCriteria("Property3", DateTime, true, false, false, Nil, None, None),
+      MetadataCriteria("Property4", Integer, true, false, false, Nil, None, None)
+    )
+    val closureMetadataCriteria = MetadataCriteria("ClosureType", Boolean, true, false, false, List("yes", "no"), None, Some(Map("Closed" -> dependentMetadataCriteria)), None)
+    val input = List(
+      Metadata("ClosureType", "Closed"),
+      Metadata("Property1", ""),
+      Metadata("Property2", ""),
+      Metadata("Property3", "1990-55-10T00:00:00"),
+      Metadata("Property4", "tt")
+    )
+    val metadataValidation = new MetadataValidation(closureMetadataCriteria, Nil)
+    val error = metadataValidation.validateClosureMetadata(input)
+    error should be(
+      List(
+        Error("Property1", NO_OPTION_SELECTED_ERROR),
+        Error("Property2", EMPTY_VALUE_ERROR),
+        Error("Property3", INVALID_NUMBER_ERROR_FOR_MONTH),
+        Error("Property4", NUMBER_ONLY_ERROR)
+      )
+    )
+  }
+
+  "validateClosureMetadata" should "return an error if dependent closure metadata has invalid values" in {
+
+    val dependentMetadataCriteria = List(
+      MetadataCriteria(
+        "Property1",
+        Boolean,
+        true,
+        false,
+        false,
+        List("yes", "no"),
+        dependencies = Some(Map("yes" -> List(MetadataCriteria("Property11", Text, true, false, false, Nil))))
+      ),
+      MetadataCriteria("Property2", Text, true, false, false, Nil, None, None),
+      MetadataCriteria("Property4", Integer, true, false, false, Nil, None, None)
+    )
+    val closureMetadataCriteria = MetadataCriteria("ClosureType", Boolean, true, false, false, List("yes", "no"), None, Some(Map("Closed" -> dependentMetadataCriteria)), None)
+    val input = List(
+      Metadata("ClosureType", "Closed"),
+      Metadata("Property1", "yes"),
+      Metadata("Property11", ""),
+      Metadata("Property2", ""),
+      Metadata("Property4", "")
+    )
+    val metadataValidation = new MetadataValidation(closureMetadataCriteria, Nil)
+    val error = metadataValidation.validateClosureMetadata(input)
+    error should be(
+      List(
+        Error("Property1", EMPTY_VALUE_ERROR),
+        Error("Property2", EMPTY_VALUE_ERROR),
+        Error("Property4", EMPTY_VALUE_ERROR)
+      )
+    )
+  }
+
+  "validateClosureMetadata" should "not return an error if a closure metadata dependent by a descriptive metadata which is empty" in {
```

Comment:
Reword test slightly: `... if closure metadata is dependent on descriptive metadata which is empty"`